forc-test/src/error_codes.rs

```diff
@@ -0,0 +1,32 @@
+use thiserror::Error;
+
+/// Revert with this value for a failing call to `std::revert::require`.
+pub const FAILED_REQUIRE_SIGNAL: u64 = 18446744073709486080;
+
+/// Revert with this value for a failing call to `std::token::transfer_to_address`.
+pub const FAILED_TRANSFER_TO_ADDRESS_SIGNAL: u64 = 18446744073709486081;
+
+/// Revert with this value for a failing call to `std::message::send_message`.
+pub const FAILED_SEND_MESSAGE_SIGNAL: u64 = 18446744073709486082;
+
+/// Revert with this value for a failing call to `std::assert::assert_eq`.
+pub const FAILED_ASSERT_EQ_SIGNAL: u64 = 18446744073709486083;
+
+/// Revert with this value for a failing call to `std::assert::assert`.
+pub const FAILED_ASSERT_SIGNAL: u64 = 18446744073709486084;
```

Comment:
We should declare these using hex, the same way they're declared in Sway code, e.g.

```suggestion
pub const FAILED_ASSERT_SIGNAL: u64 = 0xffff_ffff_ffff_0004;
```